src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -204,7 +204,7 @@ namespace OpenMS
          map_it != input_maps.end(); ++map_it)
     {
       max_intensity = max(max_intensity, map_it->getMaxInt());
-      max_mz = max(max_mz, map_it->getMax()[0]);
+      max_mz = max(max_mz, map_it->getMax()[1]);
```

Comment:
looks good, especially as RangeManager code looks like

```
126     /// Returns the maximum intensity
127     double getMaxInt() const
128     {
129       return int_range_.maxPosition()[0];
130     }
131 
```

@andreott this if course only works if `RangeManager<2>` is used which is MSExperiment, FeatureMap and ConsensusMap -- so it seems that there is issue here to use `[1]`